docs/auto-instrumentation-install.md

```diff
@@ -30,10 +30,28 @@ If a cert-manager is not available in the cluster (or other TLS certificate sour
 using `certmanager.enabled=true`. The cert-manager issues TLS certificates the operator requires. You can use the
 commands below to run these steps.
 
-_Note: In order for trace telemetry data to be properly ingested, the attribute `deployment.environment` must be on board
-the exported traces. There are two ways to set this attribute._
-- _Use the values.yaml optional `environment` configuration_
-- _Use the Instrumentation spec with the env var OTEL_RESOURCE_ATTRIBUTES_
+<details>
+<summary>Expand for extended Instrumentation configuration details</summary>
+
+- An [opentelemetry.io/v1alpha1 Instrumentation](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation)
+object is used to configure the auto-instrumentation of your applications. To successfully enable instrumentation, the
+target pod must have an Instrumentation object available.
+- When `operator.enabled=true` and `operator.instrumentation.create=true`, the helm chart deploys a
```

Comment:
`operator.instrumentation.create` is still here and a couple of other places

Reply:
updated